### ISSUE TITLE
Pre-populate cluster URL when using `teleport status`

### ISSUE THREAD

**Issue by benarent** [gravitational/teleport#7314]
### Description

**What happened**:
While prepping for a session sharing video, I noticed we don't prepopulate the proxy URL. This will make it easier to share sessions. 

```diff
[ec2-user@ip-172-31-16-25 ~]$ teleport status
User ID     : ben@goteleport.com, logged in as ec2-user from 99.13.XXX.XXX 65514 50390
Cluster Name: acme-demo.teleport.sh
Host UUID   : ae2af243-af2d-4344-8144-2599e9b8e2bf
Session ID  : 47cba465-1675-46b0-9253-c5bd8ac1ac64
- Session URL : https://<proxyhost>:3080/web/cluster/acme-demo.teleport.sh/node/ae2af243-af2d-4344-8144-2599e9b8e2bf/ec2-user/47cba465-1675-46b0-9253-c5bd8ac1ac64
+ Session URL : https://acme-demo.teleport.sh/web/cluster/acme-demo.teleport.sh/node/ae2af243-af2d-4344-8144-2599e9b8e2bf/ec2-user/47cba465-1675-46b0-9253-c5bd8ac1ac64
```

**Comment by webvictim** [comment]
Same thing as https://github.com/gravitational/teleport/issues/3217

**Comment by zmb3** [comment]
Fixed in #18579
